wordle/src/main/kotlin/wordle/domain/Game.kt

```diff
@@ -0,0 +1,39 @@
+package wordle.domain
+
+class Game(
+    wordsReader: WordsReader,
+    answerExtractor: AnswerExtractor = FixedAnswerExtractor(),
+    private val ioProcessor: IOProcessor
+) {
+
+    private val answer = answerExtractor.extract(wordsReader.read())
+
+    private var judgements = mutableListOf<Judgement>()
```

Comment:
```suggestion
    private val answer: Word = answerExtractor.extract(wordsReader.read())

    private var judgements: MutableList<Judgement> = mutableListOf()
```

프로퍼티 타입 정도는 명시하는게 안헷갈릴듯??

Reply:
이게 IDEA에서 다 표기가 되니깐 지우게 되더라구요 ㅋㅋ 물론 팀 컨벤션을 따르겠지만 보통 프로퍼티 타입 정도는 명시하는 편인가요?

Reply:
저는 프로퍼티로는 명시하는 편